### ISSUE TITLE
Diners Club Brand not interpreted properly

### ISSUE THREAD

**Issue by jshor** [gustavofrizzo/CreditCardValidator#5]
Thanks for this package

I tried the Diners Club number _30569309025904_ but the `BrandName` returned _Unknown_ (though it ran through Luhn successfully). I could not find a sample Diners Club card number that began with "36."

It looks like the rules for 14-digit Diners Club could be consolidated to be simply:

```
                    new Rule
                    {
                        Lengths = new List<int> {14,16},
                        Prefixes = new List<string> { "300", "301", "302", "303", "304", "305", "3095", "36", "38" }
                    }
```

https://github.com/gustavofrizzo/CreditCardValidator/blob/master/Src/CreditCardValidator.Net35/CreditCardData.cs#L90

Sample credit cards: https://www.paypalobjects.com/en_US/vhelp/paypalmanager_help/credit_card_numbers.htm

**Comment by n1schal** [comment]
Any progress on this one?

**Comment by gustavo-frizzo** [comment]
Fixed at 7120448d00c1f9e4e702282f7a81ae49c9e5a2f8
Nuget Version 1.4.5.

Thanks!
